_docs/life_of_intern.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: "Life of an Intern at Puppet"
+author: dishakareer
+categories:
+  - Week 1 Update 
+  - Week 2 Update
+  - Week 3 Update 
+  - Week 4 Update 
+tags:
+ - intern
+---
+
+![DishasBlogPicture](images/mypicture.jpg)
+
+
+## Welcome! 
+
+## Life of an Intern at Puppet
+
+Hello Everyone!
+
+Welcome to my Blog Page **"Life of an Intern at Puppet"**
+
+I am sure you cannot wait to hear about all the exciting things Puppet offers for their interns!!
+
+Why not start with Who Am I and how did I got into Puppet for my placement year?
+
+I am Disha Kareer, originally from India but living in UK from last 15 years, long time isn't?
+
+I started off my academics with Knockbreda High School. This is where I completed my GCSE's and then decided to
+study for my A-Levels in Newtownbreda High School. I am very hardworking, diligent and caring individual who always had a keen interest in the field of Computing and IT. I developed a strong interest in this career path since the age of 10. This was because I always enjoyed developing and designing things
+
+I started my Higher National Diploma (HND) in Computing and Systems Development and passed the course with amazing results. I began my university journey in September 2019 to study BSc Computing and Information Technology with a Year of Professional Experience as a direct entry second year student, because I had completed a two year HND Diploma with Belfast Metropolitian College, Millfield Campus.
+
+Now, I am sure you must have understood how I landed in Puppet for my placement year, no?
+
+Well, I must say going to university was surely a life changing experience because I met and still meeting new people even today, I really enjoyed my second year, but hated the global pandemic!
+
+I spent the summer of 2019 thoroughly researching for IT companies who would be interested in placement students. Don't ask me how many companies I contacted! Why? As, I completely forgot the number :)
+
+After so many interviews, assessment days, applications and attending placement fairs, I finally landed into Puppet for my interview!
+
+Did I hear someone saying... was I nervous?
+
+Well, the answer is YES, but the team was so welcoming and made the interview process so relaxing and informal that all the nerves and stress disappeared in the air!
+
+Oh, got another question?
+
+**What questions were asked during the interview?**
+	
+_The questions were generic and the team was keen to find out what kind of person I was?_
+
+___***I would also like to add, that make sure that you bring in some IT related projects that you have developed during university, college or school, just to demonstrate your skill set and your interests during the interview.***___
```

Comment:
If you were looking to _**strongly emphasize**_ your text, you can do so with `_**` and `**_` to save yourself a few characters.